examples/arNFT_gltf_brave_robot_example.html

```diff
@@ -24,14 +24,19 @@
 				document.addEventListener('getThreejsRenderer', (ev) => {			
 				    let renderer = ev.detail.renderer.renderer
 				    renderer.outputEncoding = THREE.sRGBEncoding;
-				    renderer.physicallyCorrectLights = true;
+					renderer.physicallyCorrectLights = true;
+					console.log('setting texture outputEncoding and physicallyCorrectLights');
 				})
-				document.addEventListener('getInitThreejsRenderer', (ev) => {
+				document.addEventListener('onInitThreejsRendering', (ev) => {
 					console.log(ev.detail);
+					const renderer = ev.detail.renderer;
+					console.log(renderer);
+					const scene = ev.detail.scene;
+					console.log(scene);
+					let directionalLight = new THREE.DirectionalLight('#fff', 0.4);
+					directionalLight.position.set(0.5, 0, 0.866);
+					scene.add(directionalLight)
```

Comment:
I can also add the camera here, just to show that is possibele to get the camera too.
Something like:

```javascript
const camera = ev.detail.camera

// do something with camera...

```

Reply:
This was done.